toolchain/mfc/args.py

```diff
@@ -81,10 +81,11 @@ def add_common_arguments(p, mask = None):
     test.add_argument("-f", "--from",         default=test_cases[0].get_uuid(), type=str, help="First test UUID to run.")
     test.add_argument("-t", "--to",           default=test_cases[-1].get_uuid(), type=str, help="Last test UUID to run.")
     test.add_argument("-o", "--only",         nargs="+", type=str, default=[], metavar="L", help="Only run tests with specified properties.")
-    test.add_argument("-r", "--relentless",   action="store_true", default=False, help="Run all tests, even if multiple fail.")
+    test.add_argument("-r", "--relentless",   action="store_true", default=True, help="Run all tests, even if multiple fail.")
     test.add_argument("-a", "--test-all",     action="store_true", default=False, help="Run the Post Process Tests too.")
     test.add_argument("-%", "--percent",      type=int, default=100, help="Percentage of tests to run.")
-    test.add_argument("-m", "--max-attempts", type=int, default=3, help="Maximum number of attempts to run a test.")
+    test.add_argument("-i", "--ci-mode",      action="store_true", default=False, help="Toggles CI mode.")
```

Comment:
Why have `--ci-mode` when you can just pass `--max-attempts 3` ?

Reply:
good catch, again. I would recommend just having max-attempts and getting rid of ci-mode, which seems a bit silly.